### DISCUSSION TITLE
i can't see the hands

### DISCUSSION THREAD

**Discussion by larsthomasmedia** [mozilla/hubs#3012] [Avatar Questions] [answered]
Did anybody way i can't see the hands of an avatar i made in Blender with the help of this (https://github.com/MozillaReality/hubs-avatar-pipelines/blob/master/Blender/AvatarBot/AvatarBot_base_for_export.blend) file for the rig ?

**Comment by emclaren** [comment, marked as answer]
I believe this question was answered on Discord by my colleague Dom:

"Hands are only visible in VR, you wont see them if you are using a desktop or mobile device"
